src/main/scala/net/snowflake/spark/snowflake/SnowflakeJDBCWrapper.scala

```diff
@@ -287,10 +287,30 @@ private[snowflake] class JDBCWrapper {
 
   /**
     * Returns true if the table already exists in the JDBC database.
+    * By default, if table is an unqualified table name (without schema or database name),
+    * snowflake checks table existence through a search path. For example,
+    * it checks current schema and PUBLIC schema. If the caller intends to check
+    * the table existence in current schema only. The caller needs to alter session to set
+    * search_path as $current explicitly, for example,
+    * conn.createStatement().execute("alter session set search_path='$current'")
+    * or use alternative function: tableExistsInCurrentSchema().
     */
   def tableExists(conn: Connection, table: String): Boolean =
```

Comment:
Delete this function, keep only the one below.

Reply:
I prefer to to keep it because it is used a lot in test code. It needs many test code change. it looks not necessary.

Reply:
Can you create a follow up PR to delete it. It's confusing and error prone to have one version of the function that checks a parameter and another that doesn't.

Reply:
It makes sense. I will clean it in a following PR.